spec/features/npq_separation/admin/finance/voided_declarations_spec.rb

```diff
@@ -20,7 +20,7 @@
     expect(page).to have_css("td:nth-child(1)", text: voided_declaration.id)
     expect(page).to have_css("td:nth-child(2)", text: voided_declaration.user.id)
     expect(page).to have_css("td:nth-child(3)", text: voided_declaration.declaration_type)
-    expect(page).to have_css("td:nth-child(4)", text: voided_declaration.course.name)
+    expect(page).to have_css("td:nth-child(4)", text: voided_declaration.reload.course.name)
```

Comment:
If the declaration is needing to be reloaded, presumably because data may have changed, shouldn't the first usage of the declaration on line 20 be the one doing the reload?